test/ship_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/ship.rb'
+
+class ShipTest < Minitest::Test
+  def test_it_exists
+    cruiser = Ship.new("Cruiser", 3)
+
+    assert_instance_of Ship, cruiser
+  end
+
+  def test_it_has_attributes
+    cruiser = Ship.new("Cruiser", 3)
+
+    assert_equal "Cruiser", cruiser.name
+    assert_equal 3, cruiser.length
+  end
+
+  def test_it_has_starting_health
+    cruiser = Ship.new("Cruiser", 3)
+
+    assert_equal 3, cruiser.health
+  end
+
+  def test_it_hasnt_been_hit
+    cruiser = Ship.new("Cruiser", 3)
+
+    assert_equal false, cruiser.sunk?
+  end
+
+  def test_it_has_been_hit
+    cruiser = Ship.new("Cruiser", 3)
+    cruiser.hit
+
+    assert_equal 2, cruiser.health
+    assert_equal false, cruiser.sunk?
+  end
+
+  def test_it_has_sunk
+    cruiser = Ship.new("Cruiser", 3)
+    cruiser.hit
+    assert_equal 2, cruiser.health
+    assert_equal false, cruiser.sunk?
+    cruiser.hit
+    assert_equal 1, cruiser.health
+    assert_equal false, cruiser.sunk?
+    cruiser.hit
+    assert_equal 0, cruiser.health
+    assert_equal true, cruiser.sunk?
```

Comment:
I really like how you asserted .hit  and .health multiple times for functionality